benefits/eligibility/views.py

```diff
@@ -110,12 +110,19 @@ def start(request):
     )
 
     if verifier.is_auth_required:
-        if verifier.uses_auth_verification:
+        if verifier.name == "OAuth claims via Login.gov (MST)" or "OAuth claims via Login.gov (SacRT)":
```

Comment:
@thekaveman @angela-tran Thoughts on using this approach for changing this method:

I was thinking of adding an optional string parameter to this method, of a claim (like veteran or senior), to distinguish between Login.gov (MST), Login.gov (SacRT), and VA.gov.

This method is only used 1 other time.

Orrrrr, should I create a new method entirely like, uses_claim_for that has a required parameter of a string claim, that can be used like this: uses_claim_for("veteran") and uses_claim_for("senior")?

Reply:
Lastly, we could have no method at all, and use: `verifier.auth_provider.claim == "senior"` and `verifier.auth_provider.claim == "veteran"`